packages/data-context/src/DataContext.ts

```diff
@@ -315,6 +315,7 @@ export class DataContext {
   }
 
   makeId<T extends NexusGenAbstractTypeMembers['Node']> (typeName: T, nodeString: string) {
+    // return `${typeName}:${nodeString}`
```

Comment:
should we remove this?

Reply:
Yep, was just using this in debugging, will do